INSTALL.md

```diff
@@ -0,0 +1,142 @@
+# Overview
+
+Neural Artistic Style relies on these dependencies:
+
+ * [CUDA Toolkit](https://developer.nvidia.com/cuda-downloads)
+ * [cuDNN](https://developer.nvidia.com/cudnn) v3
+ * [DeepPy](http://github.com/andersbll/deeppy), Deep learning in Python.
+ * [CUDArray](http://github.com/andersbll/cudarray), CUDA-accelerated NumPy.
+ * [Pretrained VGG 19 model](http://www.vlfeat.org/matconvnet/pretrained), choose *imagenet-vgg-verydeep-19*.
+
+The detailed installation steps are explained in the sections below.
+
+Note: This installation in mainly for GNU/Linux distributions.
+
+# Neural Artistic Style
+
+1. Download Neural Artistic Style:
+
+    ```
+    $ git clone https://github.com/andersbll/neural_artistic_style.git
+    ```
+
+# CUDA
+
+Please refer to the [Installation Guides](http://docs.nvidia.com/cuda/index.html#installation-guides) provided by nVidia.
+
+The CUDA toolkit should be installed at `/usr/local/cuda/`.
+
```

Comment:
Just wondering if there's a particular reason you advise cudNN v3? Is it okay if I use 5.1 or have you not tested it yet?